src/geometries/polytopes.jl

```diff
@@ -156,6 +156,16 @@ function angles(c::Chain)
   map(i -> ∠(vs[i - 1], vs[i], vs[i + 1]), i1:i2)
 end
 
+function (c::Chain)(t)
+  if t < 0 || t > 1
+    throw(DomainError(t, "c(t) is not defined for t outside [0, 1]."))
+  end
+  n = nvertices(c) - !isclosed(c)
+  k = max(1, ceil(Int, n * t))
```

Comment:
Shouldn't we utilize the `measure` of each segment to find `k`? Say we have a rope `r` with 2 segments along the horizontal axis, the first with `2m` length and the second with `1m` length. What should be the result of `r(0.5)`? I understand that we should pick `k=1` in this case and the final result should be a point with x coordinate at `1.5m` . Are we on the same page?

Reply:
Yeah, you are right. That makes much more sense.

Reply:
I'll implement that tomorrow (it's late in Europe).

Reply:
I rewrote the function in terms of `measure` in [28d07dc](https://github.com/JuliaGeometry/Meshes.jl/pull/1155/commits/28d07dcd6da823ce7504952aeea023f314a357f6). I think it's still not ideal because we need some quite high tolerances in the tests, but at least it should do the job despite the rounding errors.